test/windows/local_test.rb

```diff
@@ -40,6 +40,33 @@
     cmd.stderr.must_equal ''
   end
 
+  it 'uses a named pipe if available' do
```

Comment:
I don't see anything obvious in this test that actually tests that the named pipe is used. How are you asserting that in this test?

Reply:
Great point, I added words to the comments to make it more clear. Basically, it runs a PowerShell command and checks if the pipe was created by looking for the file that would be created.